### DISCUSSION TITLE
Import / export JSON settings icon ambiguous

### DISCUSSION THREAD

**Discussion by PatrickF1** [rxhanson/Rectangle#483] [General]
Hi, I love the UI overall and one small way to make it better, IMO, is to change the icons or just write out the names for import/export. I had clicked on the export one thinking it was the import one and didn't realize my mistake for a while.

**Comment by rxhanson** [comment]
Thanks for the feedback! This is something I plan to do. I was putting it off until potentially a time when more text-based UI changes would make it in. That way, there's less churn on localizing the app if multiple things are localized at once instead of piecemeal.

**Reply by rxhanson** [reply]
Went ahead and rolled this into v0.47, out shortly.

**Reply by PatrickF1** [reply]
Thank you!
